stable/sentry/templates/workers-deployment.yaml

```diff
@@ -113,9 +113,18 @@ spec:
         - mountPath: /etc/sentry
           name: config
           readOnly: true
+        - mountPath: {{ .Values.persistence.filestore_dir }}
```

Comment:
Let's follow CamelCase practice.

Reply:
I didn't change the name. (it was added on initial sentry commit https://github.com/helm/charts/commit/cb0060835763bcdd68fb080ab4799f36a1033fa1) I just added the same volume to the worker deployments

Reply:
Always leave the code cleaner then how you found it? :)

Reply:
I'm on it :)

Reply:
also the value need to be passed as `SENTRY_FILESTORE_DIR` env. I'm adding it too

Reply:
is it ok to remove this value and hardcode it? I don't see any case that changing `SENTRY_FILESTORE_DIR` from `/var/lib/sentry/files` (which is inside the container) to other values does any help

Reply:
I think in this case it's ok.